client/client.js

```diff
@@ -69,6 +105,26 @@ function serviceDestroy(name, callback) {
 }
 Client.prototype.serviceDestroy = serviceDestroy;
 
+/**
+ * Find a service.
+ *
+ * @param {string|number} serviceNameOrId Service ID or Name.
+ * @param {function} callback Callback function.
+ */
+function serviceFind(serviceNameOrId, callback) {
+  var Service = this.models.ServerService;
+  var filter = {
+    where: {
+      or: [
+        {name: serviceNameOrId},
```

Comment:
Good point. I'll keep that in mind when we start implementing multi-service PM and Mesh